docs/guide/getting-started.md

```diff
@@ -29,7 +30,7 @@ nav_order: 1
 
 1. Add Storybook Server as a dev dependency. 
    ```console
-   yarn add @storybook/server @storybook/addon-controls --dev
+   yarn add @storybook/server @storybook/addon-docs @storybook/addon-controls --dev
```

Comment:
I don't think docs are required are they?

Reply:
Hey @jonspalmer, thanks for your patience. Yep, you're right. Not sure where I was going with this.